### PR TITLE
Add SourceDrop 1.5

### PR DESCRIPTION
**Pull request by sponomarev**

### CODE REVIEW

File: Casks/sourcedrop.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'sourcedrop' do
+  version '1.5'
+  sha256 '86437846de0cb1690ab583ee1a01ce4a01b438292b84b43e7f5709eb9fa3c144'
+
+  url "https://github.com/hohl/sourcedrop-osx/releases/download/r1.5/SourceDrop-r#{version}.zip"
+  name 'SourceDrop'
+  name 'SourceDrop for OS X is a PasteBin client which allows you to easily share your snippets by drag and drop.'
```
Comment (vitorgalvao) [inline Casks/sourcedrop.rb]: This line should be removed.

**Comment by sponomarev** [comment]
Fixed and rebased over latest master

File: Casks/sourcedrop.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'sourcedrop' do
+  version '1.5'
+  sha256 '86437846de0cb1690ab583ee1a01ce4a01b438292b84b43e7f5709eb9fa3c144'
+
+  url "https://github.com/hohl/sourcedrop-osx/releases/download/r1.5/SourceDrop-r#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/sourcedrop.rb]: Just noticed there’s an extra instance of `1.5` that should be changed to `#{version}`.

**Comment by sponomarev** [comment]
Fixed. And thank you for your patient :smiley_cat:

**Comment by vitorgalvao** [comment]
Thank you for the submission.

**Pull request merged**